### PR TITLE
Synced some dungeon symbol names with Sky

### PR DESCRIPTION
**Pull request by AnonymousRandomPerson**

### CODE REVIEW

**Review by SethBarberee — CHANGES_REQUESTED** [review]

**Comment by AnonymousRandomPerson** [comment]
This will probably conflict with https://github.com/pret/pmd-red/pull/255. I'll let that PR merge first before this one.

File: src/dungeon_ai_item_weight.c [hunk 1]
```diff
@@ -219,7 +219,7 @@ u32 GetAIUseItemProbability(Entity *targetPokemon, Item *item, u32 itemTargetFla
             }
             break;
         case ITEM_CHERI_BERRY:
-            if (pokemonInfo->nonVolatile.nonVolatileStatus != STATUS_PARALYSIS)
+            if (pokemonInfo->burnClassStatus.status != STATUS_PARALYSIS)
```
Comment (SethBarberee) [inline src/dungeon_ai_item_weight.c:222]: So burn class has both paralysis and poison? This name doesn't make a lot of sense.. where does this come from in sky? Debug strings?

File: src/dungeon_move.c [hunk 2]
```diff
@@ -3227,11 +3227,11 @@ bool8 CanMonsterUseMove(Entity *pokemon, Move *move, bool8 hasPPChecker)
         {
             return FALSE;
         }
-        if (pokemonInfo->volatileStatus.volatileStatus == STATUS_TAUNTED && !MoveIgnoresTaunted(move))
+        if (pokemonInfo->cringeClassStatus.status == STATUS_TAUNTED && !MoveIgnoresTaunted(move))
         {
             return FALSE;
         }
-        if (pokemonInfo->volatileStatus.volatileStatus == STATUS_ENCORE)
+        if (pokemonInfo->cringeClassStatus.status == STATUS_ENCORE)
```
Comment (SethBarberee) [inline src/dungeon_move.c:3234]: Similar comment here on why cringe is for taunt and encore..

**Review by SethBarberee — CHANGES_REQUESTED** [review]

File: src/dungeon_ai_item_weight.c (hunk 1 above)
Comment (AnonymousRandomPerson) [reply src/dungeon_ai_item_weight.c:222]: The Sky names are from pmdsky-debug (i.e., documented by their hacking community), and are based on the status with an ID of 1 for the respective status slot. I never liked the names I gave to these status variables as a lot of them are inaccurate or confusing. For example, "non-volatile" status" in the main games includes both sleep and freeze as well, which are not grouped together with burn/paralysis/poison in Mystery Dungeon. IMO the statuses are so varied that it's hard to come up with good names for them, so I prefer the Sky community's more orderly approach.

Comment (SethBarberee) [reply src/dungeon_ai_item_weight.c:222]: Gotcha. Still don't like it but least there's a little explanation for it.

**Review by SethBarberee — CHANGES_REQUESTED** [review]
Needs rebase/conflicts fixed but otherwise fine with me.

**Comment by AnonymousRandomPerson** [comment]
Conflicts resolved.

**Review by SethBarberee — APPROVED** [review]

**Pull request merged**